src/main/views/search-option.njk

```diff
@@ -17,13 +17,13 @@
             name: "find-choice",
             fieldset: {
                 legend: {
-                    text: 'Find a court or tribunal list',
+                    text: 'Do you know the name of the court or Tribunal?',
```

Comment:
Only minor however is it correct that Tribunal starts with a capital and court doesn't?

I'm thinking both should be lower case